generate-key.js

```diff
@@ -21,7 +40,7 @@ async function exportKey(key) {
     )
 }
 
-var key = await generateKey();
+var key = await generateKey("your password", "your salt", 10000)
```

Comment:
Why not random salt like this: https://github.com/mdn/dom-examples/blob/master/web-crypto/derive-key/pbkdf2.js#L65 ?

Reply:
I thought that it would be more easier for people when they need to pass the informations instead than the key.

We just need to send a message or saying the "passphrase" and "salt", but I don't know if th salt have a limited size for pbkdf2 ?

Reply:
Salt should be random otherwise if you let the choice, people with enter "salt" and that defeat the purpose of a salt. 

Maybe we could print the final result in base64

Reply:
If they let "salt" but I think that we could generate the key directly from password and salt in the "keystorage" so it could avoid to send the key.

If we randomize the salt it could be hard when we need to give it to another person without using something like an email

Reply:
let's use this solution